### PR TITLE
ci/feat(seed-build): add darwin build

### PR DESCRIPTION
**Pull request by yqlbu**
### Background

Add build support for `darwin_amd64` and `darwin_arm64`

### Checklist

- [x] The Pull Request has been fully tested
- [ ] There's an entry in the CHANGELOGS
- [ ] There is a user-facing docs PR against https://github.com/daeuniverse/dae

### Full changelogs

- feat(seed-build): add darwin build
- ci(friendly-filenames): update darwin friendly build name
- patch(seed-build): ONLY copy systemd service if goos == 'linux'

### Issue reference

NA

### Test Result

NA

### CODE REVIEW

**Comment by mzz2017** [comment]
What about windows?

**Comment by yqlbu** [comment]
> What about windows?

Will do it later. Let's do one at a time.

File: install/friendly-filenames.json [hunk 1]
```diff
@@ -11,5 +11,7 @@
   "linux-mips64": { "friendlyName": "linux-mips64" },
   "linux-mipsle": { "friendlyName": "linux-mips32le" },
   "linux-mips": { "friendlyName": "linux-mips32" },
-  "linux-riscv64": { "friendlyName": "linux-riscv64" }
+  "linux-riscv64": { "friendlyName": "linux-riscv64" },
```
Comment (mzz2017) [inline install/friendly-filenames.json:14]: Shall we use macos as friendlyName?
Refer to https://github.com/v2fly/v2ray-core/blob/master/release/friendly-filenames.json

**Review by mzz2017 — CHANGES_REQUESTED** [review]

File: install/friendly-filenames.json (hunk 1 above)
Comment (yqlbu) [reply install/friendly-filenames.json:14]: Good catch. Resolved in https://github.com/juicity/juicity/pull/11/commits/115fc4dc1686656dbefabe633a46b1b41eabe97e

**Review by mzz2017 — DISMISSED** [review]

File: install/friendly-filenames.json [hunk 2]
```diff
@@ -11,5 +11,7 @@
...
+  "darwin-amd64": { "friendlyName": "macos-64" },
```
Comment (mzz2017) [inline install/friendly-filenames.json:15]: x86_64

Keep the same with linux

**Review by mzz2017 — CHANGES_REQUESTED** [review]

File: install/friendly-filenames.json (hunk 2 above)
Comment (yqlbu) [reply install/friendly-filenames.json]: Resolved in [atch(friendly-filenames): update darwin default build name](https://github.com/juicity/juicity/pull/11/commits/599b51834fa2c18bd169e1a436c1b143afd09e1c)

**Review by mzz2017 — APPROVED** [review]

**Comment by mzz2017** [comment]
Test Result:

image

**Pull request merged**